src/main/java/org/takes/misc/Href.java

```diff
@@ -185,6 +189,7 @@ public Href path(final String suffix) {
         return new Href(
             URI.create(
                 new StringBuilder(this.uri.toString())
+                    .append(SLASH)
```

Comment:
Looks like a redundant constant: http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants